### PR TITLE
Upgrading to NDK R10e and gcc 4.9

### PR DESCRIPTION
**Pull request by kneth**
Upgrading to NDK R10e and gcc 4.9.

Superseed #1937 

@realm/java

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,8 @@
+0.87.1
+ * Upgraded to NDK R10e. Using gcc 4.9 for all architectures.
+ * Updated Realm Core to 0.95.6
+   - Fixed a bug where an async query can be copied incomplete in rare cases.
```
Comment (cmelchior) [inline changelog.txt]: Missing issue number

**Comment by cmelchior** [comment]
```
A problem occurred evaluating project ':realm-jni'.
> Your NDK version: r10e-rc4. Realm JNI should be compiled with the version r10e of NDK.
```

Weird CI reports a release candidate as the installed NDK version?

**Comment by cmelchior** [comment]
Apart from small comments :+1:

**Comment by emanuelez** [comment]
retest this please

**Comment by emanuelez** [comment]
retest this please

**Comment by emanuelez** [comment]
Retest this please

**Comment by nhachicha** [comment]
can we create an alias for `r10e-rc4` the local installation of NDK is `r10e`

**Comment by nhachicha** [comment]
:+1:

**Comment by beeender** [comment]
:+1:

**Comment by kneth** [comment]
:+1:

**Pull request merged**